controls/sshd_spec.rb

```diff
@@ -465,7 +465,7 @@
   case os[:family]
   when 'debian' then
     describe sshd_config do
-      its('DebianBanner') { should eq('no') }
+      its('DebianBanner') { should eq(nil) }
```

Comment:
@samjmarshall we have support for debian 8-10 (all non-EOL versions), I guess this change is valid only for Debian 10. Can you maybe elaborate a bit?

Reply:
I checked that with Debian 10 and I still see the DebianBanner option in sshd_config man page. Do I miss something?

Reply:
Thanks @artem-sidorenko I don't think you're missing anything at all. I think this change is only valid for Ubutnu.